omod-common/src/main/java/org/openmrs/module/webservices/rest/web/RestUtil.java

```diff
@@ -831,13 +831,18 @@ public static SimpleObject wrapErrorResponse(Exception ex, String reason) {
 		StackTraceElement[] steElements = ex.getStackTrace();
 		if (steElements.length > 0) {
 			StackTraceElement ste = ex.getStackTrace()[0];
+			String stackTraceDetailsenabled_gp = Context.getAdministrationService()
```

Comment:
Can you take a look at our variable naming conventions? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-NamingConventions